src/components/Pagination/leftArrow.js

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-export default function LeftArrow() {
+export default function LeftArrow({ className }) {
     return (
         <svg
-            fill="#576574"
+            fill="currentColor"
```

Comment:
@TahimiLeonBravo check this color change

Reply:
@rafaelmg83 let's leave these icons with dark gray color